src/App/navigation/routes/unauthorizedStackRoutes.ts

```diff
@@ -1,48 +1,35 @@
-import React from "react";
-import DrawerNavigator from "../navigators/DrawerNavigator";
 import { EmailAuthScreen } from "@screens/EmailAuthScreen";
 import { PhoneAuthScreen } from "@screens/PhoneAuthScreen";
 import { OTPVerification } from "@screens/OTPVerification";
 import { StackNavigationOptions } from "@react-navigation/stack";
-import { AuthorizedStackParamList, UnauthorizedStackParamList, ROUTES } from "@shared/api";
-import { NativeStackScreenProps } from "react-native-screens/native-stack";
+import { UnauthorizedStackParamList, ROUTES } from "@shared/api";
 
 type RouteConfig<ParamList extends Record<string, object | undefined>, RouteName extends keyof ParamList> = {
     name: RouteName;
-    component: React.ComponentType<NativeStackScreenProps<ParamList, RouteName>>;
+    component: React.ComponentType<any>; // Expect a component, not an element
```

Comment:
what about this any, can you change this on some certain type?

Reply:
nothing but <any> fits here